tasks/release.py

```diff
@@ -293,14 +293,34 @@ def tag_devel(ctx, release_branch, commit="HEAD", push=True, force=False):
 
 
 @task
-def finish(ctx, release_branch, upstream="origin"):
+def finish(ctx, release_branch, release_date=None, upstream="origin"):
```

Comment:
nit - in general you should avoid adding parameters between two others. If someone relies on the order, either they're screwed or you'll have to update their call. In this case it does not change much but you you should add it at the end